Source/WebCore/html/HTMLFieldSetElement.cpp

```diff
@@ -87,11 +87,12 @@ static void updateFromControlElementsAncestorDisabledStateUnder(HTMLElement& sta
     it.dropAssertions();
 
     while (it != range.end()) {
-        if (auto* listedElement = it->asValidatedFormListedElement())
+        RefPtr element = *it;
```

Comment:
Should this be `Ref element`? Given `element` is not null checked in the next line, it seems like it.